src/runner.go

```diff
@@ -72,9 +67,11 @@ func verifyYamlFile(yamlData []byte) bool {
 
 	output, err := cmd.Output()
```

Comment:
Should we add the removed log about verifying `log.Infoln("Verifying signature ...")` here? Or maybe immediately after the gpg condition?

Reply:
Moved that log to be up top, but it will print only when the gpg check is enabled. otherwise, it will be a different log.